PWGHF/D2H/DataModel/ReducedDataModel.h

```diff
@@ -183,16 +169,39 @@ auto invMassDplusToPiKPi(const T& pVec0, const T& pVec1, const T& pVec2)
                                  RecoDecay::getMassPDG(kKPlus),
                                  RecoDecay::getMassPDG(kPiPlus)});
 }
-} // namespace hf_cand_3prong_reduced
+} // namespace hf_charm_cand_reduced
 
-DECLARE_SOA_TABLE(HfCand3ProngReduced, "AOD", "HFCAND3PRONGRED", //! Table with 3prong candidate information for reduced workflow
+DECLARE_SOA_TABLE(HfRed2Prongs, "AOD", "HFRED2PRONGS", //! Table with 2prong candidate information for reduced workflow
+                  o2::soa::Index<>,
+                  hf_track_index_reduced::Prong0Id, hf_track_index_reduced::Prong1Id,
+                  hf_track_index_reduced::HfRedCollisionId,
+                  HFTRACKPAR_COLUMNS,
+                  hf_cand::XSecondaryVertex, hf_cand::YSecondaryVertex, hf_cand::ZSecondaryVertex,
+                  hf_charm_cand_reduced::InvMassD0, hf_charm_cand_reduced::InvMassD0Bar,
+                  aod::track::Px<aod::track::Signed1Pt, aod::track::Snp, aod::track::Alpha>,
+                  aod::track::Py<aod::track::Signed1Pt, aod::track::Snp, aod::track::Alpha>,
+                  aod::track::Pz<aod::track::Signed1Pt, track::Tgl>);
+
+DECLARE_SOA_TABLE(HfRed2ProngsCov, "AOD", "HFRED2PRONGSCOV", //! Table with 2prong candidate covariance for reduced workflow
+                  o2::soa::Index<>,
+                  HFTRACKPARCOV_COLUMNS,
+                  o2::soa::Marker<1>);
+
+DECLARE_SOA_TABLE(HfRed3Prongs, "AOD", "HFRED3PRONG", //! Table with 3prong candidate information for reduced workflow
```

Comment:
Maybe add a `S` at the end of the table description to be consistent with `HFRED2PRONGS`

Reply:
I think that the convention is to put the `s` in the first argument and not in the third one (see e.g. https://github.com/AliceO2Group/AliceO2/blob/90c5ac72684d34daa85cb024ba7cdb1e0f126c0a/Framework/Core/include/Framework/AnalysisDataModel.h#L314), so I would let it like this if you agree

Reply:
Indeed ! I also ended up doing it like this. In this spirit, we'll need to remove the `s` in `HFCAND2PRONGS`.

Actually, if we want to use `DECLARE_SOA_INDEX_COLUMN_FULL` (with an index pointing to a table), we need the table to be defined as follows: `DECLARE_SOA_TABLE(Name, "AOD", Description)` with `Name = Description + 's'`. This requires to redefine the name of `HfRedTrackBase` in `HfRedTrackBases`, otherwise the merger is looking for the name of the table (with the last letter removed): `hfredtrackbas` in our case (and it creates an error).